src/components/applicationForm/uploadDocument.tsx

```diff
@@ -0,0 +1,55 @@
+import { FormHeader } from "../formSections"
+
+const requiredDocuments = {
+    resume: true,
+    portfolio: true,
+    cv: true
```

Comment:
can we try implementing this as 1 document first, then whatever new document is uploaded will replace the old one?